github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -126,6 +182,21 @@ periodics:
       - "/bin/bash"
       - "-c"
       - >
+        # For getting centos image tag from s390x prow job and use same for x86 and manifest-list images
```

Comment:
I would prefer if the s390x job could relay on the tag from the x86 image rather than relying on the s390x job here.

Reply:
Updated with first having x86 built then s390x along with publishing multi-arch manifest